taxonomies/categories.txt

```diff
@@ -1230,6 +1243,12 @@ food_groups:en: en:Alcoholic beverages
 pnns_group_2:en:Alcoholic beverages
 nova:en:3
 
+<en:Alcoholic beverages
+en:Drinks Flavoured - Ready to Drink
```

Comment:
Those kind of categories make me a little skeptic. 
Should we try renaming them ? 
Do we have a integrate first, rename later approach ? 
Do we keep them as is for all eternity ?

Reply:
I agree. I want to take it step by step, so later renaming and better integration